### ISSUE TITLE
Add Questions, Answers, and Comments to /stats/range/__ pages

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#1667]
Hi, this is a [first-timers-only issue](https://github.com/publiclab/plots2/projects/2). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

If you have contributed before, consider leaving this one for someone new, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

## The problem

When you go to a page like:

https://publiclab.org/stats/range/10-05-2015/10-05-2016/

You see something like:

```
744 notes in this period
3358 wiki edits in this period
31286 new member registrations in this period
```

We want to add 3 more stats to that page, including:

* Questions
* Answers
* Comments

****

## Solution

This page is generated here: https://github.com/publiclab/plots2/blob/master/app/controllers/stats_controller.rb#L14-L16

And displayed on this template: https://github.com/publiclab/plots2/blob/master/app/views/stats/range.html.erb#L9

**Note:** if solving this, please choose just one of the above to submit a change for; we'll run through this three times, once for each type.

### Questions

For Questions, you can make a query to collect them like this (i think):

```ruby
    @questions = Node.questions
                     .select(%i[created type status])
                     .where(type: 'note', status: 1, created: @start.to_i..@end.to_i)
                     .count
```

Then add a line to the template (linked above) to print that out, like:

```html
<h4><%= @questions %> questions posted in this period</h4>
```

### Answers

For answers, try:

```ruby
    @answers = Answer.where(type: 'note', status: 1, created: @start.to_i..@end.to_i)
                     .count
```

Then add a line to the template (linked above) to print that out, like:

```html
<h4><%= @answers %> answers posted in this period</h4>
```

### Comments

For comments, try:

```ruby
    @comments = Comment.where(type: 'note', status: 1, created: @start.to_i..@end.to_i)
                        .count
```

Then add a line to the template (linked above) to print that out, like:

```html
<h4><%= @comments %> comments posted in this period</h4>
```

## Steps to Fix

- [ ] claim this issue with a comment here, below, and ask any clarifying questions you need
- [ ] set up a repository locally following the README instructions, and make sure that all tests pass
- [ ] try to fix the issue following the steps above, but even before you're done, you can:
- [ ] commit your changes and start a pull request (see [contributing to Public Lab software](https://publiclab.org/wiki/contributing-to-public-lab-software)) but mark it as "in progress" if you have questions or if you haven't finished
- [ ] alert someone via the developers list (see below) to have your pull request merged. This may take a few extra steps depending on when you do it, but we'll help you out!

Please email the developers list (see https://publiclab.org/wiki/developers) or go to [the chatroom](http://publiclab.org/chat) if you have questions, and take a look at [our first-timers landing page for more information](https://github.com/publiclab/plots2/projects/2)!

**Comment by MelvillesMobyDick** [comment]
I'm a first-timer and would like to give this one a shot!

**Comment by jywarren** [comment]
Yes please! We'd love your contribution and would be happy to help out!

**Comment by sagarpreet-chadha** [comment]
Hey @jywarren sir...can i try this ?

**Comment by jywarren** [comment]
Yes please! we'd be happy to help as you give it a try. Thank you!!

**Comment by kaid7** [comment]
Has someone already completed this? I'd like to give this a shot.

**Comment by amitsin6h** [comment]
I was looking for first timer issue can I work on this. @jywarren

**Comment by jywarren** [comment]
Solved, i believe, in #1672! Thank you!
